src/BayesFilters/src/StateModel.cpp

```diff
@@ -8,3 +8,9 @@ Eigen::MatrixXf StateModel::getJacobian()
 {
     throw std::runtime_error("ERROR::STATEMODEL::GETJACOBIAN\nERROR:\n\tMethod not implemented.");
 }
+
+
+Eigen::VectorXf StateModel::transitionProbability(const Eigen::Ref<const Eigen::MatrixXf>& prev_states, Eigen::Ref<Eigen::MatrixXf> cur_states)
```

Comment:
Since we have `getJacobian` maybe we could call it `getTransitionProbability`. What do you think about it?

Reply:
Agree on it.

Reply:
Perfect! 👍